docs/conf.py

```diff
@@ -17,29 +17,46 @@
 # If your extensions are in another directory, add it here. If the directory
 # is relative to the documentation root, use os.path.abspath to make it
 # absolute, like shown here.
-#sys.path.append(os.path.abspath('some/directory'))
+
+on_rtd = os.getenv('READTHEDOCS') == 'True'
+
+sys.path.insert(0, os.path.abspath('..'))
```

Comment:
os.path.dirname(os.path.abspath(__file__))

Reply:
```
os.path.dirname(os.path.abspath(__file__))
```

Reply:
I also need to delete `on_rtd = os.getenv('READTHEDOCS') == 'True'` that was only for testing purposes